src/main/com/intellij/lang/jsgraphql/schema/library/GraphQLLibraryTypes.java

```diff
@@ -37,6 +37,18 @@ public boolean isEnabled(@NotNull Project project) {
         }
     };
 
+    public static GraphQLLibraryDescriptor APOLLO_KOTLIN = new GraphQLLibraryDescriptor("APOLLO_KOTLIN") {
+        @Override
+        public boolean isEnabled(@NotNull Project project) {
+            return GraphQLSettings.getSettings(project).isApolloKotlinSupportEnabled();
+        }
+
+        @Override
+        public @NotNull String getPresentableText() {
+            return GraphQLBundle.message("graphql.library.apollokotlin");
```

Comment:
Looks like you forgot to add this key "graphql.library.apollokotlin" to the localization bundle.